docs/docs/60-new-docs/50-user-guide/20-how-to-guides/15-working-with-freight.md

```diff
@@ -213,3 +260,33 @@ kargo get freight \
     }
 ]
 ```
+
+</TabItem>
+</Tabs>
+
+## Promoting a Freight to a Stage
+
+<Tabs groupId="promoting-freight-to-a-stage">
+<TabItem value="ui" label="Using the UI" default>
+
+1. Click the three dots on the `Freight` in the <Hlt>Freight Timeline</Hlt>,
+    then select <Hlt>Promote</Hlt>:
+
+    ![Promote Freight to a Stage](img/freight-promotion.png)
+
+2. Select the `Stage` and click <Hlt>Promote</Hlt>:
+
+    ![Promote Freight to a Stage](img/freight-promotion-2.png)
+
+</TabItem>
+
+<TabItem value="cli" label="Using the CLI">
+
+To promote `Freight` to a `Stage` using the CLI, run:
+
+```shell
+kargo promote --stage <stage> --project <project> --freight <freight>
```

Comment:
As with the other pull request, I believe `freight-alias` might be easier for people.